### PR TITLE
Remove `issue_body` from YAML issue forms

### PR DESCRIPTION
**Pull request by lukehefson**
- [ ] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/HEAD/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/brew/blob/HEAD/Library/Homebrew/test/PATH_spec.rb).
- [ ] Have you successfully run `brew style` with your changes locally?
- [ ] Have you successfully run `brew typecheck` with your changes locally?
- [ ] Have you successfully run `brew tests` with your changes locally?

-----

`issue_body` is now an invalid key (in favour of using WYSIWYG `textarea`s). So `issue_body: false` needs to be removed completely in order for the forms to work 🤗.

> Screenshot 2021-04-20 at 17 03 52

### CODE REVIEW

**Review by issyl0 — APPROVED** [review]
Thanks for your first contribution to Homebrew, @lukehefson! Where do you see the message about this key being deprecated? I can't immediately find it on eg. https://github.com/Homebrew/brew/blob/master/.github/ISSUE_TEMPLATE/feature.yml. Are there any docs we can read about what's valid?

**Comment by issyl0** [comment]
Going to rebase this branch on the latest `master` to fix CI after #11199.

**Review by issyl0 — APPROVED** [review]
Re-approving now CI will go green.

**Comment by carlocab** [comment]
Thanks @lukehefson!

**Comment by lukehefson** [comment]
> Where do you see the message about this key being deprecated? I can't immediately find it on eg. https://github.com/Homebrew/brew/blob/master/.github/ISSUE_TEMPLATE/feature.yml.

@issyl0 Not sure why you didn't see the warning message there at the time 😕 

> Are there any docs we can read about what's valid (apart from on the side of the web UI editor) - and the recent changes?

As issue forms is a private alpha we have a private feedback repo where we invite Maintainers who are in the alpha to give feedback and where we announce changes. I'm not sure offhand who from Homebrew is in the feedback repo (as the invite would have been before my time), but if you or anyone else would like to be in it let me know!

**Comment by carlocab** [comment]
Before this change, in https://github.com/Homebrew/brew/blob/44188a3f6eb1497ef0d0954fc69c234ba0082f84/.github/ISSUE_TEMPLATE/bug.yml, I see

Screenshot 2021-04-21 at 10 10 39

**Comment by lukehefson** [comment]
> Before this change, in https://github.com/Homebrew/brew/blob/44188a3f6eb1497ef0d0954fc69c234ba0082f84/.github/ISSUE_TEMPLATE/bug.yml, I see

Ah yes sorry, that's the difference between when we had a deprecation _warning_ (in the lead up to yesterday's deprecation) as opposed to the validation _error_ when we officially deprecated `issue_body` yesterday.

**Comment by carlocab** [comment]
Ah, yes. The image I show above does look slightly different from what I recalled seeing before, as I was pretty sure I remembered an orange/yellow-ish warning.

**Pull request merged**
